aten/src/ATen/xpu/detail/XPUHooks.cpp

```diff
@@ -25,7 +25,13 @@ std::string XPUHooks::showConfig() const {
 
 int32_t XPUHooks::getGlobalIdxFromDevice(const at::Device& device) const {
   TORCH_CHECK(device.is_xpu(), "Only the XPU device type is expected.");
+#ifdef _WIN32
+  TORCH_CHECK(
+      false,
+      "Default context is not supported on XPU on Windows. So we can NOT find its global index of the ATen device.");
```

Comment:
What is using this function? What is the impact of forbidding this?

Reply:
This function is mainly used for dlpack. We want to inform users that dlpack doesn't support XPU on Windows, since we can't ensure context consistency because the default context is not being used on Windows for XPU.